sashimi/hardware/light_source/interface.py

```diff
@@ -14,34 +15,40 @@ class LaserWarning(Warning):
     pass
 
 
-class AbstractLightSource:
+class AbstractLightSource(ABC):
     def __init__(self, port):
         self.port = port
         self._status = LaserState.OFF
         self.intensity_units = None
 
-    def start(self):
+    def start(self):  #TODO: remove?
         pass
-
+    
+    @abstractmethod
     def set_power(self, current):
         """Sets power of laser based on self.intensity and self.intensity_units"""
         pass
```

Comment:
Similarly to my previous comment:

I think that this method is redundant since what aims to do is taken care of by the `intensity property setter` and the `set_current` method (see line 33 of the [cobolt.py ](https://github.com/portugueslab/sashimi/blob/95732a0a35de7485c900c5d599a05673452c6a20/sashimi/hardware/light_source/cobolt.py#L33) )

Reply:
Probably in this case the `set_current` method should be refactored in set_power no? @diegoasua ?

Reply:
I agree. This was left behind because there was a plan to add a calibration function that would translate from laser intensity to user-friendly units, hence power. The user should not see amperes in the GUI because... unless one is familiar with the setup that does not have any physical meaning, as it depends on a **calibration curve**. Optimally there would be two methods, one communicating with the laser, hence setting the current, one calculating power on sample from a calibration table.